docs/pages/configuration_files.rst

```diff
@@ -96,6 +96,10 @@ config file. Each site can overwrite any of these variables.
   ``https_proxy: http://user:pass@addr:port``. If no proxy is specified, PRAW
   will pick up the environment variable for https_proxy, if it has been set.
 
+* *validate_certs:* A **boolean** to indicate if SSL certificates should be
```

Comment:
Can you move this section to just after `timeout`? The `http_proxy` sections should also be in a sorted location (feel free to move those too if you want).